wp-parsely.php

```diff
@@ -768,6 +768,28 @@ public function insert_parsely_page() {
 		return $parsely_page;
 	}
 
+	/**
+	 * Compare the post_status key against an allowed list (by default, only 'publish'ed content includes tracking data)
+	 *
+	 * @param  int|WP_Post $post  Which post object or id to check.
+	 * @return bool               Should the post status be tracked for the provided post's post_type. By default, only 'publish' is allowed.
+	 *
+	 * @see wp_parsely_track_status_slugs
+	 * @since 2.5.0
+	 */
+	public static function post_has_trackable_status( $post ) {
+		/**
+		 * Filters the status slugs that are permitted to be tracked.
+		 * By default, the only slug allowed is 'publish'. Override this list if your post type necessitates.
+		 *
+		 * @since 2.5.0
+		 *
+		 * @param string[]    $status_slugs The list of post_status slugs that are allowed to be tracked.
+		 * @param int|WP_Post $post         Which post object or id is being checked.
+		 */
+		$status_slugs = apply_filters( 'wp_parsely_track_status_slugs', array( 'publish' ), $post );
+		return in_array( get_post_status( $post ), $status_slugs, true );
```

Comment:
```suggestion
		 * @param string[]    $trackable_statuses The list of post statuses that are allowed to be tracked.
		 * @param int|WP_Post $post               Which post object or ID is being checked.
		 */
		$statuses = apply_filters( 'wp_parsely_trackable_statuses', array( 'publish' ), $post );
		return in_array( get_post_status( $post ), $statuses, true );
```

Despite there being a `get_post_stati()` function, I think `statuses` is more accessible to folks.

Removed references to "status slugs" as it's the terminology used in WP - there's no "slug" for it - the key is just is the "post status" (label is a separate thing).